src/wrapper.rs

```diff
@@ -529,6 +529,18 @@ impl Dxc {
             .result()?;
         Ok(DxcLibrary::new(library))
     }
+
+    pub fn create_reflector(&self) -> Result<DxcReflector> {
+        let mut reflector: ComPtr<IDxcContainerReflection> = ComPtr::new();
+
+        self.get_dxc_create_instance()?(
```

Comment:
Unrelated: we cache (keep alive) the `Library` but might as well store `Symbol<DxcCreateInstanceProc>` too instead of `dlsym`'ing it every time.

Reply:
I'll create an additional PR that caches the symbol.